lib/src/screens/debug/scheme_management/requestor_scheme_detail_screen.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_i18n/flutter_i18n.dart';
+
+import '../../../models/irma_configuration.dart';
+import '../../../models/scheme_events.dart';
+import '../../../theme/theme.dart';
+import '../../../widgets/irma_app_bar.dart';
+import '../../../widgets/irma_icon_button.dart';
+import '../../../widgets/irma_repository_provider.dart';
+import '../util/snackbar.dart';
+
+class RequestorSchemeDetailScreen extends StatelessWidget {
+  final RequestorScheme requestorScheme;
+
+  const RequestorSchemeDetailScreen({super.key, required this.requestorScheme});
+
+  void _onDeleteScheme(BuildContext context) {
+    IrmaRepositoryProvider.of(context).bridgedDispatch(RemoveRequestorSchemeEvent(
+      schemeId: requestorScheme.id,
+    ));
+    Navigator.of(context).pop();
+
+    if (!context.mounted) return;
+    showSnackbar(
+      context,
+      FlutterI18n.translate(
+        context,
+        'debug.scheme_management.remove',
+        translationParams: {
+          'scheme': requestorScheme.id,
+        },
+      ),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = IrmaTheme.of(context);
+    return Scaffold(
+      appBar: IrmaAppBar(
+        title: requestorScheme.id,
+        actions: [
+          IrmaIconButton(
+            icon: Icons.delete,
+            onTap: () => _onDeleteScheme(context),
+          )
+        ],
+      ),
+      body: SingleChildScrollView(
+        padding: EdgeInsets.all(theme.screenPadding),
+        child: Column(
+          children: [
+            ListTile(
+              title: const Text('Type'),
+              subtitle: Text(requestorScheme.demo ? 'Demo requestor scheme' : 'Production requestor scheme'),
```

Comment:
Noticed we do not use translated values for `title` and `subtitle` in ListTile. Same applies to the already existing screen`lib/src/screens/debug/scheme_management/scheme_manager_detail_screen.dart`. 
Since it's debug functionality it's not a big issue and we can leave it as is, but wanted to mention it anyway.

Reply:
True, but the translation wouldn't be that much different in Dutch and English. Only 'Production' would differ.

Reply:
Agreed. Then let's leave it as is.